biz.aQute.bndlib/bnd.bnd

```diff
@@ -62,6 +63,12 @@ Import-Package: \
 
 -maven-scope: provided
 
+-maven-dependencies.core: \
```

Comment:
This is still wring. You don't need this since bndlib itself contains the package for non-OSGi use cases. Doing thing just adds a second copy of the package to the classpath for maven/gradle.

Also, you need to readd the Import-Package statement for org.osgi.dto. Even though bndlib itself does not currently use the org.osgi.dto package, it does no harm to import it and it prevents us from making silly mistakes in the future when we actually do start to use it in bndlib and then forget to add the Import-Package for it.